dbt_subprojects/hourly_spellbook/macros/models/_sector/rollup_economics/check_mapping_hash.sql

```diff
@@ -0,0 +1,42 @@
+{% macro check_mapping_hash(settlement_layer) %}
```

Comment:
Ideally we may add an macro to check if the spell need to be refreshed, also this saves the common code blocks